src/coreclr/src/vm/comcallablewrapper.cpp

```diff
@@ -3719,78 +3556,6 @@ void ComMethodTable::LayOutClassMethodTable()
     LOG((LF_INTEROP, LL_INFO1000, "LayOutClassMethodTable: %s, parent: %s, this: %p  [DONE]\n", m_pMT->GetDebugClassName(), pParentClass ? pParentClass->GetDebugClassName() : 0, this));
 }
 
-
-BOOL ComMethodTable::CheckSigTypesCanBeLoaded(MethodTable *pItfClass)
-{
-    CONTRACTL
-    {
-        THROWS;
-        GC_TRIGGERS;
-        MODE_ANY;
-        PRECONDITION(CheckPointer(pItfClass));
-    }
-    CONTRACTL_END;
-
-    if (!IsLayoutComplete())
-    {
-        if (!IsSigClassLoadChecked())
```

Comment:
Looks like `IsSigClassLoadChecked`, `SetSigClassLoadChecked`, `IsSigClassCannotLoad`, and `SetSigClassCannotLoad` can be removed too.